src/main/java/org/kairosdb/datastore/cassandra/CassandraDatastore.java

```diff
@@ -201,6 +212,15 @@ public void putDataPoint(String metricName,
                              ImmutableSortedMap<String, String> tags,
                              DataPoint dataPoint,
                              int ttl) throws DatastoreException {
+
+        Span parentSpan = tracer.scopeManager().active().span();
+        Tracer.SpanBuilder spanBuild = tracer.buildSpan("putDataPoint").withTag("Name", "putData");
+
+        if (parentSpan !=null)
```

Comment:
Please format your code
`parentSpan !=null` ==> `parentSpan != null`

For reference: https://www.jetbrains.com/help/idea/editor-basics.html